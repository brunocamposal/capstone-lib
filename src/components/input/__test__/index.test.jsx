import React from 'react'
import Input from '../index'
import renderer from 'react-test-renderer'

it('input renderizando corretamente', () => {
  const tree = renderer
    .create(
      <Input
        id='id'
        name='name'
        onChange={() => {}}
        placeholder='place'
        value='value'
        type='text'
      />
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
