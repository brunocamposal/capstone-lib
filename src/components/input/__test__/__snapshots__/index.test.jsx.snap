// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input renderizando corretamente 1`] = `
<input
  id="id"
  name="name"
  onChange={[Function]}
  placeholder="place"
  type="text"
  value="value"
/>
`;
